app/appmessage/message.go

```diff
@@ -156,6 +156,9 @@ const (
 	CmdVirtualDaaScoreChangedNotificationMessage
 	CmdGetBalancesByAddressesRequestMessage
 	CmdGetBalancesByAddressesResponseMessage
+	CmdNotifyNewBlockTemplateRequestMessage
```

Comment:
What about StopNotify?

Reply:
It's not implemented for most events. But we should add it eventually.